hedera-mirror-rest/controllers/contractController.js

```diff
@@ -699,7 +700,7 @@ class ContractController extends BaseController {
       throw new NotFoundError();
     }
     const contract = rows[0];
-    if (contract.file_id !== null) {
+    if (contract.file_id !== null && isValidUserFileId(contract.file_id)) {
```

Comment:
change `isValidUserFileId` to first do `_.isNil`, then this line can be simplified to
```suggestion
    if (utils.isValidUserFileId(contract.file_id)) {
```